modules/hosted-che/partials/ref_about-hosted-che.adoc

```diff
@@ -7,19 +7,19 @@
 = About Eclipse Che hosted by Red Hat
 
 Eclipse Che hosted by Red Hat is an open source product based on link:https://www.eclipse.org/che/[Eclipse {prod-short}] that is running on link:https://www.openshift.com/products/dedicated/[OpenShift Dedicated].
-The new service is part of the link:https://developers.redhat.com/developer-sandbox[Developer Sandbox for Red Hat OpenShift] offering, and is using link:https://developers.redhat.com/products/codeready-workspaces/overview[CodeReady Workspaces], which is built upon Eclipse Che and is optimized for Red Hat OpenShift and Red Hat Linux.
+The new service is part of the link:https://developers.redhat.com/developer-sandbox[Developer Sandbox for Red Hat OpenShift] offering, and is using link:https://developers.redhat.com/products/devspaces/overview[Red Hat OpenShift Dev Spaces], which is built upon Eclipse Che and is optimized for Red Hat OpenShift and Red Hat Linux.
 
-== Differences between Eclipse Che and CodeReady Workspaces
+== Differences between Eclipse Che and Red Hat OpenShift Dev Spaces
 
-Red Hat CodeReady Workspaces is the product that is built from the Eclipse Che project. The product is normally two versions behind the project. Red Hat also provides licensing, packaging, and support, so CodeReady Workspaces is considered a more stable product than the upstream Eclipse Che project. See link:https://access.redhat.com/documentation/en-us/red_hat_codeready_workspaces/2.14/html/release_notes_and_known_issues/about-codeready-workspaces_crw[About Red Hat CodeReady Workspaces].
+Red Hat Red Hat OpenShift Dev Spaces is the product that is built from the Eclipse Che project. The product is normally two versions behind the project. Red Hat also provides licensing, packaging, and support, so Red Hat OpenShift Dev Spaces is considered a more stable product than the upstream Eclipse Che project. See link:https://access.redhat.com/documentation/en-us/red_hat_devspaces/2.14/html/release_notes_and_known_issues/about-devspaces_crw[About Red Hat Dev Spaces].
```

Comment:
By using `{prod-short}` you will avoid typos like this. There is no Red Hat Dev Spaces. There is only Red Hat OpenShift Dev Spaces (or shorter name, OpenShift Dev Spaces).

Reply:
Also if you're going to change the link from 2.14 ... you likely want to update to 3.0 as the suggested new URL is invalid for 2 reasons.

1. there's no `red_hat_devspaces` doc, it's `red_hat_openshift_dev_spaces`. 
2. there's no 2.y version of `red_hat_openshift_dev_spaces` - it starts at version 3.0.

New docs will be at: https://access.redhat.com/documentation/en-us/red_hat_openshift_dev_spaces/3.0/

Reply:
Thank you so much!